frontend/relay-workflows-lib/lib/components/TemplatesList.tsx

```diff
@@ -22,41 +23,78 @@ const templatesListQuery = graphql`
 
 export default function TemplatesList() {
   const data = useLazyLoadQuery<TemplatesListQuery>(templatesListQuery, {});
-  const templates = data.workflowTemplates.nodes;
+  const [search, setSearch] = useState("");
+  const [filteredTemplates, setFilteredTemplates] = useState(
```

Comment:
This doesn't need to be a state because it's essentially derived from another state `search`